src/Windows/Markdown/Editor/Impl/Preview/Code/EvalSession.cs

```diff
@@ -43,7 +43,7 @@ internal sealed class EvalSession : IDisposable {
 
             if (!RSession.IsHostRunning) {
                 var settings = _services.GetService<IRSettings>();
-                await RSession.EnsureHostStartedAsync(new RHostStartupInfo(), SessionCallback, 3000, ct);
+                await RSession.EnsureHostStartedAsync(new RHostStartupInfo(isInteractive: true), SessionCallback, 3000, ct);
```

Comment:
@karthiknadig , @int19h , do you remember what are the side effects of making session interactive?